test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -162,4 +162,23 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
       AccountRootFlag.LSF_REQUIRE_DEST_TAG,
     )
   })
+
+  it('requireDestinationTags/allowNoDestinationTag - rippled', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    // GIVEN an existing testnet account
+    // WHEN requireDestinationTags is called, followed by allowNoDestinationTag
+    await issuedCurrencyClient.requireDestinationTags(wallet)
+    const result = await issuedCurrencyClient.allowNoDestinationTag(wallet)
+
+    // THEN both transactions were successfully submitted and there should be no flag set on the account.
+    await XRPTestUtils.verifyFlagModification(
+      wallet,
+      rippledGrpcUrl,
+      result,
+      AccountRootFlag.LSF_REQUIRE_DEST_TAG,
+      false,
+    )
+  })
```

Comment:
once you've added the integration test for enabling Require Destination, followed by a failed txn submission with a missing destination, maybe you can add a test immediately afterward that uses the same wallet (which we *just* verified had the requireDest flag set) and then un-set it, and successfully send a payment without a dest tag?